client/src/components/qAndA/qAndAList.jsx

```diff
@@ -1,20 +1,29 @@
 import React from 'react';
 import Question from './question.jsx';
 
-const createList = (data) => {
-  if (data !== undefined) {
-    let list = data.map((item) =>
-    <div>
-      <div className="qAndA-list-item" >Q: {item}</div>
-      </div>
-    );
-    return list;
+const createAnswersList = (data) => {
+  let answersData = [];
+  for (var key in data) {
+    answersData.push(data[key]);
   }
+  let result = answersData.map((answer) =>
+    <div>
+      <div className="qAndA-list-answer-item" >A: {answer.body}</div>
+    </div>
+  );
+  return result;
 };
 
 const QAndAList = ({questions}) => (
   <div className="qAndA-list">
-    <div>{createList(questions)}</div>
+    {console.log('questions obj inside qanda: ', questions)}
+    {questions.length !== 0 ? questions.results.map((item) =>
+      <div className="qAndA-list-question-item">
+        Q: {item.question_body}<br />
+        by {item.asker_name} | {item.question_date}
+        {createAnswersList(item.answers)}
```

Comment:
If you want, you could create a new component called AnswerList and pass the answers array to it. Then it will only be concerned with displaying answers.

Reply:
Agreed, good idea - I've pushed the changes to this pull request